samples/wifi/radio_test/single_domain/README.rst

```diff
@@ -8,9 +8,68 @@ The sample shows how to use the radio test subcommands to configure the paramete
 
 The Wi-Fi Radio test also supports programming the Factory Information Configuration Registers (FICR) fields defined in the nRF7002 one-time programmable (OTP) memory.
 
+.. contents::
+   :local:
+   :depth: 2
```

Comment:
Move this before line 6.

Change line 6 "The Wi-Fi Bluetooth LE Radio test sample" to "The Wi-Fi Bluetooth LE Radio test (Single domain) sample"
```suggestion
.. contents::
   :local:
   :depth: 2
```